maltd-frontend/src/components/GeneralButton/GeneralButton.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from "react";
+import { Button } from "reactstrap";
+import PropTypes from "prop-types";
+import "./GeneralButton.css";
+
+export default function GeneralButton({
+  generalButton: { type, color, disabled, block, active, outline, placeholder }
+}) {
+  return (
+    <Button
+      className="general-button"
+      type={type}
+      color={color}
+      disabled={disabled}
+      block={block}
+      active={active}
+      outline={outline}
+    >
+      {placeholder}
```

Comment:
this isn't really a placeholder, it is more of an actual display text value - so maybe we can call this prop something like `displayText`?